history-of-the-quakers/original/29-book-3-chapter-3.adoc

```diff
@@ -218,46 +245,45 @@ to which he replied, "`I love and honour all magistrates and rulers,
 who are for the punishment of evil doers, and the praise of them that do well.`"
 Then secretary Rawson called out, Edward Wharton come to the bar.
 
-Edward Wharton.
+_Edward Wharton._
 Yea, and to the bench too, for thou hast no evil to lay to my charge.
-Secretary.
+
+_Secretary._
 Hold up your hand.
 
-Edw.
-Wharton.
+_Edw. Wharton._
 I will not, thou hast no evil to charge me with.
 
-Secretary.
+_Secretary._
 Edward Wharton, hear your sentence of banishment.
 
-Edw.
-Wharton.
+_Edw. Wharton._
 Have a care what you do, for if you murder me, my blood will lie heavy upon you.
 
-Secretary.
+_Secretary._
 Edward Wharton attend to your sentence of banishment:
 You are upon pain of death to depart this jurisdiction,
 it being the 11th of this instant March, by the one and twentieth of the same,
 on the pain of death.
 
-Edw.
-Wharton.
+_Edw. Wharton._
 I am a single man, and I have dealings with some people;
 it were good I had time to make clear with all, and then if you have power to murder me,
 you may.
 
 Then the governor and secretary laid their heads together.
 
-Governor.
+_Governor._
 If we should give him a hundred days, it is all one.
 
-Edw.
-Wharton.
+_Edw. Wharton._
 Nay, I shall not go away, therefore be careful what you do.
 
 Then addressing the people assembled in the court, he spoke audibly as followeth, viz.
 
-"`All people, take notice, what horrible, wicked and unjust men these are;
+[quote]
+____
```

Comment:
I'm not 100% sure how i feel marking this up as a [quote] -- i see why you did it, and it might be the right move.  but i wonder, if you're changing a bunch of these sections to discourse-part -- i wonder it might feel more natural then to leave it not marked up...